### PR TITLE
 JAVA class

### PR DESCRIPTION
**Pull request by varun1118**
I have added JAVA Class

### CODE REVIEW

File: src/Varun.java [hunk 1]
```diff
@@ -0,0 +1,5 @@
+public class Varun {
+    public static void main(String[] args) {
+
```
Comment (AlokRevi) [inline src/Varun.java:3]: print your name

Comment (Sudhakarm18) [inline src/Varun.java:3]: fix empty class

**Review by Sudhakarm18 — APPROVED** [review]

File: src/Varun.java (hunk 1 above)
Comment (yadhuykr) [reply src/Varun.java:3]: add more features

**Review by AlokRevi — APPROVED** [review]

File: src/Varun.java (hunk 1 above)
Comment (rickymangla) [inline src/Varun.java:3]: add some functions

**Review by rickymangla — APPROVED** [review]

File: src/Varun.java (hunk 1 above)
Comment (prateekahuja9) [inline src/Varun.java:3]: why an empty class?

Comment (SonamMehla) [inline src/Varun.java:3]: add some methods

**Pull request merged**